### DISCUSSION TITLE
Executable for Windows users

### DISCUSSION THREAD

**Discussion by wahlp** [AnthonyMichaelTDM/DRG-Save-Editor#67] [General]
was recommending this tool to friend, but they had so much trouble with the setup process that found it easier to edit the file for them. it seems that historically, lots of issues are from users who cannot seem to setup their environment for python or install the packages.

if we can remove the need for these users to go through the whole setup, there would be far fewer opened github issues over such a problem.

@AnthonyMichaelTDM what are your thoughts on including an executable in Releases? i have been trying to make one with pyinstaller and it seems to work: [link](https://github.com/wahlp/DRG-Save-Editor/releases/tag/v2024.07.08.214225)

**Comment by AnthonyMichaelTDM** [comment]
If you can make a functioning executable, open a pr and I’ll review it.

**Reply by wahlp** [reply]
opened #68
